### ISSUE TITLE
Test using Lighthouse

### ISSUE THREAD

**Issue by anthkris** [anthkris/mws-restaurant-stage-2#3]
Follow the recommendations provided by Lighthouse to achieve the required performance targets: 

- Progressive Web App score should be at 90 or better.
- Performance score should be at 70 or better.
- Accessibility score should be at 90 or better.

You can audit your site's performance with Lighthouse by using the Audit tab of Chrome Dev Tools.
